Accessibility modifications

Thoth1111 opened the pull request:
### Accessibility Report:
We checked the live demo for accessibility issues by running checks on the following:

1.  Page titles - No issues found.
2.  Image text alternative - 
https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L39-L43, 
https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L46, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L57, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L70, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L96, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L109, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L122, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L146, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L156, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L167, https://github.com/Thoth1111/Alwyn_Portfolio/blob/master/index.html#L196-L200
3. Text headings - No issues found.
4.  Color contrasts - No issues found.
5.  Resize - No issues found.
6.  Interactions: No issues found.
7. Moving content - No issues found.
8.  Multi-media - No issues found.
9.  Basic structure of the page - No issues found.

Commit:
Add alt-text to images

joshualyngdoh committed:
Updated README file

OyePriscilla reviewed (changes requested):
Hi @Thoth1111  and @joshualyngdoh 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425615-8927e377-c543-407b-8da4-5c09fecc261b.gif)

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

- [x] Please, according to the evaluation guidelines, we need to run your live demo through [this tool](https://www.aditus.io/button-contrast-checker/). Not all of the checks are passing for your project, kindly run your live demo through it and consider making all the changes suggested by the tool.
- [x] Please kindly add an action and method to your form element, please also ensure to add validations to the input elements as well. All these will enhance the accessibility to your project; you may want to reference [this requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). 👍 
- [x] Kindly change the header tag in line 31 to `h1 `for SEO optimization. 💁 

> Note: I would have added these comments with GH inline comments  but because you worked on the issues raised in previous milestone, I couldn't add inline comments to the concerned changes. Kindly go through the points highlighted above. Thanks. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Modify buttons

Commit:
Merge branch 'Accessibility-modifications' of github.com:Thoth1111/Alwyn_Portfolio into Accessibility-modifications

Commit:
Modify form with action and method

Thoth1111 commented:
### Changes made

We've made the following changes:

- Included the action and method to the form element and the validations.
- Modified line 31 to a h1 tag.
- Changed the color of the buttons to pass the contrast ratio check.

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @Thoth1111 and @joshualyngdoh , 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://camo.githubusercontent.com/4799731d86dd0d52a0bb7e22af48077bd16556ee26061146e987e9a98d36ee0d/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f594b646d3931584f4e4c5461772f67697068792d646f776e73697a65642d6d656469756d2e676966">

## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
